CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 1.4.0 - Release
+  - Improve error handling, group switching (Stretch).
+
+## 1.3.0 - Only released in alpha/beta
+  - Initial support for Stretch (v2/v3) including tests
+  - Support gzip encoding
```

Comment:
Force gzip encoding, work-around for aiohttp-error

Reply:
Fixed